api/api.go

```diff
@@ -279,6 +280,11 @@ func procCreateNetwork(c libnetwork.NetworkController, vars map[string]string, b
 	processCreateDefaults(c, &create)
 
 	options := []libnetwork.NetworkOption{}
+	if len(create.NetworkOpts) > 0 {
+		if _, ok := create.NetworkOpts[netlabel.Isolated]; ok {
```

Comment:
the keyword `isolated` doesn't seem to be the right one.
The networks are by default `isolated` from one another. The actual functionality that this PR provides is disabling external connectivity / default-gateway. I guess we should call it `defaultGateway` or `external` or something equivalent ? Given the fact that we might make the `DefaultGateway` configurable with its own plugin, we have to start thinking this in that direction.